util/rom-ext-manifest-generator.py

```diff
@@ -108,22 +111,20 @@ def generate_defines(fields):
                 regions.append((field['name'], region))
             current_offset_bytes += size_bytes
 
-    return [regions, offsets]
+    return (regions, offsets)
```

Comment:
Just out of curiosity, why this change?

Reply:
I feel it's more idiomatic python to use a tuple when you're returning two things that aren't conceptually a sequence of the same kinds of things - here they're a pair of separate items.